modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/enums/AndesConfiguration.java

```diff
@@ -560,7 +560,12 @@
      */
     PERFORMANCE_TUNING_MESSAGE_EXPIRATION_BATCH_SIZE
             ("performanceTuning/messageExpiration/messageBatchSize", "1000", Integer.class),
-
+    /**
```

Comment:
we need a better explanation that this. could you ellaborate more on
1) how stuff works,
2) How that relates to configuration